.eslintrc

```diff
@@ -3,6 +3,7 @@
   "parser": "babel-eslint",
   "env": { "jasmine": true },
   "rules": {
+    "object-curly-newline": 0,
```

Comment:
unrelated to this PR, but I agree with this change. IMO splitting those object destructures you made into multiple lines would be less readable anyways!